source/elements/oneMath/source/architecture/other_architecture.inc.rst

```diff
@@ -2,19 +2,19 @@
 ..
 .. SPDX-License-Identifier: CC-BY-4.0
 
-.. _onemkl_arch_other:
+.. _onemath_arch_other:
 
 Other Features
 ----------------
-This section covers all other features in the design of oneMKL architecture.
+This section covers all other features in the design of oneMath architecture.
 
 
-.. _onemkl_spec_current_version:
+.. _onemath_spec_current_version:
 
 Specification Version and Compliance
 +++++++++++++++++++++++++++++++++++++++++++++
 
-Each oneMKL domain must define a preprocessor macro to represent the version of the specification that the implementation is compliant with.
+Each oneMath domain must define a preprocessor macro to represent the version of the specification that the implementation is compliant with.
 
 The macros for each domain are listed as follows:
```

Comment:
Shall we also rename the macros `ONEMKL_<domain>_SPEC_VERSION` here?

Reply:
Thank you for your comment @rafbiels.
I updated macros in [5149a8c](https://github.com/uxlfoundation/oneAPI-spec/pull/596/commits/5149a8c0b058eddcdfd1b0d68434193e29089dd0)